ibm/service/eventnotification/resource_ibm_en_destination_custom_email.go

```diff
@@ -231,6 +302,12 @@ func resourceIBMEnCustomEmailDestinationUpdate(context context.Context, d *schem
 
 	options.SetInstanceID(parts[0])
 	options.SetID(parts[1])
+	verifyCustomEmailDestinationConfiguration.SetType(d.Get("verification_type").(string))
```

Comment:
why we need to add this here if you are adding as part of if d.HasChange("verification_type")